src/Equinox.MessageDb/Tracing.fs

```diff
@@ -4,33 +4,20 @@ module internal Equinox.MessageDb.Tracing
 open Equinox.MessageDb.Core
 open System.Diagnostics
 
-[<AbstractClass; Sealed; System.Runtime.CompilerServices.Extension>]
-type ActivityExtensions private () =
+type private Activity with
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddExpectedVersion(act: Activity, version) =
-        match version with StreamVersion v -> act.AddTag("eqx.expected_version", v) | Any -> act
+    member act.AddExpectedVersion(version) =
+        match version with StreamVersion v -> act.SetTag("eqx.expected_version", v) | Any -> act
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddLastVersion(act: Activity, version: int64) =
-        act.AddTag("eqx.last_version", version)
+    member act.AddLastVersion(version: int64) = act.SetTag("eqx.last_version", version)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddBatchSize(act: Activity, size: int64) =
-        act.AddTag("eqx.batch_size", size)
+    member act.AddBatchSize(size: int64) = act.SetTag("eqx.batch_size", size)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddBatches(act: Activity, batches: int) =
-        act.AddTag("eqx.batches", batches)
+    member act.AddBatches(batches: int) = act.SetTag("eqx.batches", batches)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddStartPosition(act: Activity, pos: int64) =
-        act.AddTag("eqx.start_position", pos)
+    member act.AddStartPosition(pos: int64) = act.SetTag("eqx.start_position", pos)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddLoadMethod(act: Activity, method: string) =
-        act.AddTag("eqx.load_method", method)
+    member act.AddLoadMethod(method: string) = act.SetTag("eqx.load_method", method)
 
```

Comment:
can the constants live here too please (or just have a method that delegates to this and makes it private)

Reply:
as in constants for the `method`